data/presets/amenity/townhall/barangay.json

```diff
@@ -0,0 +1,25 @@
+{
+    "locationSet": {
+        "include": [
+            "ph"
+        ]
+    },
+    "icon": "fas-building-flag",
```

Comment:
Did you choose this one on purpose to be different from the icon of the "generic" town hall preset?

Reply:
I did, but I misunderstood the need for this line, so I'll just remove that.

Reply:
I'm sorry, this was supposed to be just a question. Now, the preset has no icon at all. :see_no_evil:  I'll restore the icon while merging this PR :blush: